DFC.App.MatchSkills.Services.Dysac/DysacService.cs

```diff
@@ -16,27 +21,71 @@ public class DysacService : IDysacSessionReader, IDysacSessionWriter
     {
         
         private readonly IOptions<DysacSettings> _dysacSettings;
-        private readonly IRestClient _client;
+        private readonly IRestClient _restClient;
         private readonly ILogger _logger;
         private const string ResultsEndpoint = "/result/";
-        public DysacService(ILogger<DysacService> log, IRestClient client, IOptions<DysacSettings> dysacSettings)
+        private readonly ISessionClient _sessionClient;
+
+        public DysacService(ILogger<DysacService> log, IRestClient restClient, IOptions<DysacSettings> dysacSettings, ISessionClient sessionClient)
         {
             Throw.IfNull(dysacSettings, nameof(dysacSettings));
             _logger = log;
-            _client = client;
             _dysacSettings = dysacSettings;
+            _restClient = restClient ?? new RestClient();
```

Comment:
How come we need to instantiate RestClient here?
I would have assumed the DI would have handled it